lib/scout_apm/instruments/samplers/memory.ex

```diff
@@ -0,0 +1,30 @@
+defmodule ScoutApm.Instruments.Samplers.Memory do
+  def metrics do
+    value = as_duration()
+    [
+      %ScoutApm.Internal.Metric{
+        type: "Memory",
+        name: "Physical",
+        call_count: 1,
+        total_time: value,
+        exclusive_time: value,
+        min_time: value,
+        max_time: value,
+      }
```

Comment:
I created `Metric.from_sampler_value/3` to abstract this logic out of `ScoutApm.Instruments.Samplers.Memory`. See 47a422c.